corehq/apps/zapier/apps.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+
+class ZapierConfig(AppConfig):
+
+    name = 'corehq.apps.zapier'
+    verbose_name = 'Zapier'
+
+    def ready(self):
+        from .signals import receivers  # noqa
```

Comment:
http://flake8.pycqa.org/en/latest/user/ignoring-errors.html#in-line-ignoring-errors